frontend/packages/kubevirt-plugin/src/k8s/requests/vmi/actions.ts

```diff
@@ -0,0 +1,32 @@
+import { coFetch } from '@console/internal/co-fetch';
+import { resourceURL } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared';
+import { VirtualMachineInstanceModel } from '../../../models';
+import { VMIKind } from '../../../types/vm';
+
+export enum VMIActionType {
+  Unpause = 'unpause',
+}
+
+const VMIActionRequest = async (vm: VMIKind, action: VMIActionType) => {
+  const method = 'PUT';
+  let url = resourceURL(
+    {
+      ...VirtualMachineInstanceModel,
+      apiGroup: `subresources.${VirtualMachineInstanceModel.apiGroup}`,
+    },
+    {
+      ns: getNamespace(vm),
+      name: getName(vm),
+    },
+  );
+
+  url = `${url}/${action}`;
+
+  const response = await coFetch(url, { method });
+  const text = await response.text();
+
+  return text;
+};
+
+export const unpauseVM = async (vmi: VMIKind) => VMIActionRequest(vmi, VMIActionType.Unpause);
```

Comment:
shouldn't we make it more conventional?

```suggestion
export const unpauseVMI = async (vmi: VMIKind) => VMIActionRequest(vmi, VMIActionType.Unpause);
```

Reply:
Done!